deployments/mainnet/configuration.json

```diff
@@ -0,0 +1,72 @@
+{
+  "symbol": "📈USDC",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+  "baseTokenPriceFeed": "0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6",
+  "borrowMin": "1000e6",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0.000011574074074074073e15",
+    "baseBorrowSpeed": "0.0011458333333333333e15",
+    "baseMinForRewards": "1000000e6"
+  },
+  "assets": {
+    "COMP": {
+      "address": "0xc00e94cb662c3520282e6f5717214004a7f26888",
+      "priceFeed": "0xdbd020CAeF83eFd542f4De03e3cF0C28A4428bd5",
+      "decimals": "18",
+      "borrowCF": 0.65,
+      "liquidateCF": 0.7,
+      "liquidationFactor": 0.92,
+      "supplyCap": "500000e18"
```

Comment:
I guess these will be 0

Reply:
Done.

Reply:
Oh man, this is actually causes a lot of the mainnet scenarios to fail (which kind of makes sense I guess?)

Reply:
Yeah, makes a lot of sense, in fact we would hope that. So ideally when we have the real migration and proposal constraint, that one will pass, but need to think how we control/see/verify what passes is correct 🤔

Reply:
During parking lot, we decided to temporarily revert these to non-zero values to make sure all scenarios are passing. Once we are sure scenarios pass (which they are right now), we can set these values back to zero and merge.